anomalib/models/components/feature_extractors/__init__.py

```diff
@@ -3,14 +3,16 @@
 # Copyright (C) 2022 Intel Corporation
```

Comment:
Should this module be called `feature_extraction`? This would be more in line other modules under `models.components` (e.g. `dimensionality_reduction` and `sampling`)